shared/parcel.js

```diff
@@ -21,14 +21,11 @@ export function isParcel(asset) {
 }
 
 export function toParcelObject(parcelsArray, prevParcels) {
-  return connectParcels(
-    parcelsArray,
-    parcelsArray.reduce((map, parcel) => {
-      map[parcel.id] = normalizeParcel(parcel, prevParcels[parcel.id])
-      return map
-    }, {}),
-    prevParcels
-  )
+  const parcelObject = {}
```

Comment:
Is it wrong if we call it `parcelsObject`?

Reply:
the function is called `toParcelObject` that's why i used `parcelObject`. But for consistency I think we could rename `parcelsArray` to `parcelArray`, right?

Reply:
I think that would be better to rename the function to `toParcelsObject` its return an object with all the parcels as an object. And also use `parcelsObject` and what you say `parcelArray`

Clean version:

`toParcelObject` to `toParcelsObject`
`parcelObject` to `parcelsObject`
`parcelsArray` to `parcelArray`